lib/GameConstants.h

```diff
@@ -428,10 +428,7 @@ class SecondarySkill
 
 ID_LIKE_OPERATORS(SecondarySkill, SecondarySkill::ESecondarySkill)
 
-namespace EAlignment
-{
-	enum EAlignment { GOOD, EVIL, NEUTRAL };
-}
+enum class EAlignment : uint8_t { GOOD, EVIL, NEUTRAL };

```

Comment:
EAlignment *might* also benefit from Identifier class, but not sure whether it is necessary at this point.
It also not as widespread through the code as ERes, so changing it in future will be easy enough

Reply:
I do not think than this huge PR should be more huge. I think, later is better.